ml-proto/spec/float.ml

```diff
@@ -50,7 +50,8 @@ sig
   val le : t -> t -> bool
   val gt : t -> t -> bool
   val ge : t -> t -> bool
-  val zero: t
+  val zero : t
+  val bare_nan : t
```

Comment:
Did something change in this patch to require bare_nan in the float interface? This value is an implementation detail -- and in fact, it's not actually a NaN pattern until some non-zero bits get or'd in -- so it'd be nice to keep its visibility limited if possible.